### DISCUSSION TITLE
请问支持通过pull request完善学校信息吗？

### DISCUSSION THREAD

**Discussion by WendaoLee** [CollegesChat/university-information#61] [Q&A]
以及，个人建议在问卷收集页面_建议用户注明所处校区信息_，同一个学校不同校区的住宿条件是不一样的，在未收集到足够的用户信息时容易被片面信息所误导。

还有，这个项目做得挺有帮助的。

**Comment by kaixinol** [comment]
不支持，只能用問卷。
